homeassistant/components/remote/__init__.py

```diff
@@ -57,6 +57,9 @@
 SUPPORT_LEARN_COMMAND = 1
 SUPPORT_DELETE_COMMAND = 2
 
+EVENT_TYPE = "remote_event"
```

Comment:
This should the prefixed with the integration's domain
```suggestion
EVENT_TYPE = "broadlink_remote_event"
```

Reply:
@emontnemery I added it to the `remote` component with the idea that other `remote` components would be able to implement the event and have a common set of base events that would be used. If there isn't any interest in that, I could move it into the `broadlink` component and prefix as suggested.

Reply:
If this should indeed live in the `remove` integration, I think the event's data should have a common base with optional  integration specific data.